mixer/cmd/build.go

```diff
@@ -545,6 +546,7 @@ func init() {
 
 	RootCmd.AddCommand(buildCmd)
 
+	buildBundlesCmd.Flags().BoolVar(&buildFlags.noClean, "no-clean", false, "Do not wipe the image/VER directory if it exists")
```

Comment:
Is the double negative (not no-clean) worthwhile? Would it be better to have the flag be called "clean" and default to true? Maybe also be worth saying that the cleanup happens at the start?

Reply:
mmm Yeah I can change the flag name to be more like "--don't-call-this--normally". 
I like `--ignore-dirty` or `--use-existing`

Reply:
Probably just want to use --ignore-dirty so users know they are taking on a risk by keeping dirt around :)

Reply:
I'd have to agree with @icarus-sparry here. I'd prefer it to use the old state if it exists, maybe printing a notification that it is attempting to use old state for the build. If this isn't what the user wanted they can re-run and pass --clean. I don't think mixer should be wiping previous work by default, especially if that previous work is needed in some use-cases.

The warning from mixer that the clean is happening is essentially useless, since it warns right before it actually wipes the old state. The user has no way to abort that process. Consider that someone puts work into that directory for mixer to use, mixer runs and wipes it without them knowing they had to pass --ignore-dirty to keep their work.